Dockerfile.moc-xdmod

```diff
@@ -89,6 +90,14 @@ RUN chmod g+rwX /usr/bin/xdmod-openstack-hypervisor \
     && chmod -R g+rwX /etc/httpd \
     && chgrp -R 0 /etc/httpd 
 
+# openshift reporting
+# checkout https://github.com/OCP-on-NERC/xdmod-openshift-scripts
+# copy openshift xdmod script to /usr/bin/xdmod-openshift.sh (to be created)
+#   # run oc login
+#   # set OPENSHIFT_PROMETHEUS_URL
+#   # python openshift_metrics/openshift_prometheus_metrics.py --output-file <output>
+#   # /usr/bin/xdmod-shredder -f slurm -i <output> -r <resource>
+
```

Comment:
now I understand you notation. So:
1) I would set the URL in the job
2) you will need to down load the configuration files from the database as the volume storing them cannot be shared (it is RWO and not RWX as NESSE(?) does not support RWX).

Reply:
Here (https://github.com/CCI-MOC/xdmod-cntr/pull/47/files#diff-507b1af3802214396997dcfc436e9c9e62f853102bea79b1fe6e4c030d34c735).  I thought I pointed this out to you a week or so ago.

Hopefully, we get RWX volumes on OpenShift and the kludge will go away!

Reply:
Yep, you definitely did! Sorry, this comment was from a while ago.

You may see a lot of updates to this PR, only because I'm pushing updates to my local branch while testing various things out (I just managed to get the updated build config working). Feel free to ignore everything - I'll let you know when I think this is ready for another look!